src/site/content/en/blog/codelab-sign-in-form-best-practices/index.md

```diff
@@ -89,21 +90,21 @@ This doesn't work yet.
 
 The CSS layout is mobile-first:
 
-* The default CSS is for viewports less than `450px` wide.
-* The media query section sets overrides for viewports that are at least `450px` wide.
+* The default CSS is for viewports less than 450 pixels wide.
+* The media query section sets overrides for viewports that are at least 450 pixels wide.
 
-When building your own form like this, it's very important at this point to
+When building your own form like this, it's very important at this point in the process to
 test your code on real devices on desktop and mobile:
 
-* Is label and input text readable, especially for people with low vision?
+* Is label and input text readable, especially for people with poor vision?
```

Comment:
```suggestion
* Is label and input text readable, especially for people with low vision?
```
`poor` is not the preferred term according to people with this condition

Reply:
Will replace with "bad."

Reply:
"bad" is also not the correct term. It's "low".

Reply:
General rule: use the term that the people affected by this condition (or any) have chosen for themselves: https://www.afb.org/blindness-and-low-vision/eye-conditions/low-vision-and-legal-blindness-terms-and-descriptions

![image](https://user-images.githubusercontent.com/4713486/101554587-c2f43400-396b-11eb-8193-53a968069777.png)